README.md

```diff
@@ -1,2 +1,81 @@
 # cluster-guide
 A guide to using clusters in general (and th-top in particular)
+
+## Loading modules
+
+### Examples
+```shell
+$ module load anaconda-python
+```
+Load `anaconda-python`.
+
+```shell
+$ module load julia
+```
+Load default version of `julia`.
+
+```shell
+$ module load julia/1.2.0
+```
+Load `julia` v1.2.0.
+
+In order to avoid having to do that at each log in, one can append these commands to the `.bash_profile` file in the home directory. For instance:
+```shell
+$ echo "module load <module_name>" >> ~/.bash_profile
+```
+
+## Submitting jobs via `qsub`
+
```

Comment:
```suggestion
Th-top is not like a standard desktop computer, but it is a system composed by a master server managing several computing nodes. 
Running computations on such systems is a bit different than usual, and it will be explained below.
```

Reply:
have a look at the email I sent you. we need to add a few lines of context

Reply:
I now added all the information of your mail.